sync/utils/dbfs.py

```diff
@@ -0,0 +1,63 @@
+import base64
+
+from sync.clients.databricks import DatabricksClient, get_default_client
+
+
+def format_dbfs_filepath(base_path: str):
+    base_path = base_path.strip("/")
+    return f"dbfs:/{base_path}" if "dbfs:/" not in base_path else base_path
+
+
+def read_dbfs_file(
```

Comment:
Since DBFS requires some looping to read/write files over 1MB, I just packaged that up in these utility methods

Reply:
Are these methods only used in awsdatabricks? I'm a bit wary of starting "utils" packages in the root in general as it tends to end up accumulating abandoned code.